.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: enhancement
+assignees: tbaumer22
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Complaint template more like?? This is about features someone wants implemented?

Reply:
This is only relevant if their feature idea is due to a potential flaw in the design of our project.

Reply:
Updated in most recent commit.